apps/coscrad-frontend/src/components/resources/spatial-features/spatial-feature-detail.full-view.presenter.tsx

```diff
@@ -5,15 +5,22 @@ import {
     ISpatialFeatureViewModel,
 } from '@coscrad/api-interfaces';
 import { isNullOrUndefined } from '@coscrad/validation-constraints';
-import { Grid, styled } from '@mui/material';
+import { Box, Grid, styled } from '@mui/material';
+import { useState } from 'react';
```

Comment:
Be sure to update the Cypress test for the spatial feature flow. If there isn't one, create it now.